app/src/main/java/com/kangmicin/hotmovie/ui/main/MainActivity.kt

```diff
@@ -69,15 +69,15 @@ class MainActivity : AppActivity(),
         tvModel.getTvs().observe(this, Observer<List<Tv>> {initView(it)})
 
         movieModel.hasLoading().observe(this,  Observer<Boolean> {
-            if (it == true) {
+            if (it) {
                 initLoading()
             }
         } )
 
         tvModel.hasError().observe(this, Observer<Boolean> {
-            if (it == true) {
+            if (it) {
```

Comment:
why we not use `==` ?

Reply:
Because if(it) is the same thing as if(it == true) since the code will only be executed if it == true and it can only be true or false we can do if(it) directly to simplify things, also the compiler will transform the code more quickly and easily, gaining performance, since you don't need to perform a comparison between a variable and a value.